doc/source/api/solver/fieldinfo.rst

```diff
@@ -0,0 +1,98 @@
+.. _ref_field_info:
+
+Field info
+==========
+
+You can use field info objects to access Fluent field information.
+
+Accessing field info objects
+----------------------------
+
+In order to access field info, launch the fluent solver, and make field
+data available (for example, by reading case and data files):
```

Comment:
and initializing... field info is not available otherwise I think.

Reply:
Yes, Thank you @dnwillia-work. This has been done.